packages/components/modules/payments/web/hooks/useStripeHook.tsx

```diff
@@ -76,12 +117,35 @@ const useStripeHook = () => {
       mutationKey: [CONFIRM_CARD_PAYMENT_API_KEY.get()],
     })
 
+  const useGetProduct = (customerId: string) =>
+    useQuery({
+      queryKey: ['useGetProduct', customerId],
```

Comment:
Let's move this queryKey to the service/keys file and apply it as a function, the same way we did for CONFIRM_CARD_PAYMENT_API_KEY:

export const CONFIRM_CARD_PAYMENT_API_KEY = {
default: ['confirmCardPayment'] as QueryKey,
get: (...params: string[]) => [...CONFIRM_CARD_PAYMENT_API_KEY.default, ...params] as QueryKey,
}